README.md

```diff
@@ -39,13 +39,13 @@ it('should be allow to check if promise rejected', function() {
 
 ## .rejectedWith
```

Comment:
Actually that's not the real signature. It's confusing – maybe best to just link to it – http://shouldjs.github.io/#assertion-throw